pkg/helm/actions/upgrade_release_test.go

```diff
@@ -98,6 +98,81 @@ func TestUpgradeRelease(t *testing.T) {
 	}
 }
 
+func TestUpgradeReleaseWithoutChartUrl(t *testing.T) {
```

Comment:
this test could be perfectly added to other tests under `TestUpgradeRelease`, please move it there.

Reply:
Done